core/templates/pages/signup-page/signup-page.component.ts

```diff
@@ -49,7 +49,7 @@ angular.module('oppia').component('signupPage', {
       ctrl.isFormValid = function() {
         return (
           ctrl.hasAgreedToLatestTerms &&
-          (ctrl.hasUsername || !ctrl.getWarningText(ctrl.username))
+          (ctrl.hasUsername || !ctrl.warningI18nCode)
```

Comment:
Why this change?

Reply:
`ctrl.getWarningText()` doesn't exist at all. So I replaced it by intuition to `ctrl.warningI18nCode`